### ISSUE TITLE
GQL transactions filter - min block height

### ISSUE THREAD

**Issue by ppedziwiatr** [textury/arlocal#24]
Hi.

It seems that when I'm asking GQL endpoint for transactions with filter on min and max block height, the min. value is exclusive, instead of inclusive.

Eg. current block height is 5, I'm asking for transactions with min. block height = 3 and max block height = 5 - in response I'm getting transactions on block heights 4, 5 - instead of transactions on block heights 3, 4, 5.

Let's say the network info returns:
```
    {
      network: 'arlocal.N.1',
      version: 1,
      release: 1,
      queue_length: 0,
      peers: 0,
      height: 2,
      current: 'uh3m601k0kfj9c1r6i4yea79dp7fh9p9xt1d7gdg9l0',
      blocks: 2,
      node_state_latency: 0
    }
 ```
 I'm adding three new transactions:
 ```
    await contract.writeInteraction({ function: 'add' });
    await mine();
    await contract.writeInteraction({ function: 'add' });
    await mine();
    await contract.writeInteraction({ function: 'add' });
    await mine();
 ```
    
After that the network info is:
```
{   
      network: 'arlocal.N.1',
      version: 1,
      release: 1,
      queue_length: 0,
      peers: 0,
      height: 5,
      current: '4k4f6d13cq9u4pdxph1a5d1lghm03on2kt0tk9g5rno',
      blocks: 5,
      node_state_latency: 0
       }
```   

Then I'm asking for transactions with min. block height = 3 and max. block height = 5.
    
I'm getting 2 transactions as a result, instead of 3:
    `2021-09-16T21:37:06.242Z DEBUG [ContractInteractionsLoader] All loaded interactions: [ { from: 3, to: 5, loaded: 2 } ]`

**Comment by ppedziwiatr** [comment]
I've made few more tests.

This code:
![image](https://user-images.githubusercontent.com/84311757/133760783-e8b7f220-c92c-47a4-be91-dd253dfe144f.png)

gives this output:
![image](https://user-images.githubusercontent.com/84311757/133760888-ee27a83c-6e5f-4ab6-85e8-8263d720603a.png)

- where I would expect, that it would return exactly one result - in case of arweave.net and this request (current block height at the time of writing this is 771431)
![image](https://user-images.githubusercontent.com/84311757/133761091-7e5d6e95-a445-4fe3-9edf-d9d43e4d15fc.png)
- it works as expected.

**Comment by ppedziwiatr** [comment]
One last thing - if I print the full network info, I get:
![image](https://user-images.githubusercontent.com/84311757/133763472-49d7f9ce-7e2e-4831-ad75-31ba803cd145.png)

you can see, the the block `height` and `blocks` are the same (1)

In case of arweave.net gateway:
![image](https://user-images.githubusercontent.com/84311757/133763754-839ca6ab-b921-4c71-a548-ebb93922dcdf.png)
- `height` seems to be always lower by one than `blocks`

**Comment by kouraf** [comment]
fixed in #25
